extensions/blocks/google-calendar/google-calendar.php

```diff
@@ -11,64 +11,31 @@
 
 const FEATURE_NAME = 'google-calendar';
 const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
-/**
- * Check if the block should be available on the site.
- *
- * @return bool
- */
-function is_available() {
-	if (
-		defined( 'IS_WPCOM' )
-		&& IS_WPCOM
-		&& function_exists( 'has_any_blog_stickers' )
-	) {
-		if ( has_any_blog_stickers(
-			array( 'premium-plan', 'business-plan', 'ecommerce-plan' ),
-			get_current_blog_id()
-		) ) {
-			return true;
-		}
-		return false;
-	}
-
-	return true;
-}
 
 /**
  * Registers the block for use in Gutenberg
  * This is done via an action so that we can disable
  * registration if we need to.
  */
 function register_block() {
-	if ( is_available() ) {
-		jetpack_register_block(
-			BLOCK_NAME,
-			array(
-				'render_callback' => 'Jetpack\Google_Calendar_Block\load_assets',
-			)
-		);
-	}
+	jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'render_callback' => 'Jetpack\Google_Calendar_Block\load_assets',
+		)
+	);
 }
+
 add_action( 'init', 'Jetpack\Google_Calendar_Block\register_block' );
 
 /**
  * Set the availability of the block as the editor
  * is loaded
  */
 function set_availability() {
-	if ( is_available() ) {
-		\Jetpack_Gutenberg::set_extension_available( BLOCK_NAME );
-	} else {
-		\Jetpack_Gutenberg::set_extension_unavailable(
-			BLOCK_NAME,
-			'missing_plan',
-			array(
-				'required_feature' => 'google_calendar',
-				'required_plan'    => 'value_bundle',
-			)
-		);
-	}
+	\Jetpack_Gutenberg::set_extension_available( BLOCK_NAME );
```

Comment:
You can remove this altogether, no worries there! Check the map block for example.

Reply:
Done